runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkStreamingTransformTranslators.java

```diff
@@ -846,12 +846,7 @@ boolean canTranslate(
       WindowingStrategy<?, BoundedWindow> windowingStrategy =
           (WindowingStrategy<?, BoundedWindow>) input.getWindowingStrategy();
 
-      boolean hasNoSideInputs;
-      try {
-        hasNoSideInputs = CombineTranslation.getSideInputs(context.getCurrentTransform()).isEmpty();
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      }
+      boolean hasNoSideInputs = true;
 
       return windowingStrategy.getWindowFn().isNonMerging() || hasNoSideInputs;
```

Comment:
Here we can just delete the `|| hasNoSideInputs` bit, because it's now always `true`.

Reply:
Instead I just deleted the implementation of that function since it always returns true.